packages/mobile/src/verify/VerificationEducationScreen.tsx

```diff
@@ -215,6 +215,16 @@ function VerificationEducationScreen({ route, navigation }: Props) {
     }
   }
 
+  useEffect(() => {
+    if (isE2EEnv && currentState.type === StateType.EnsuringRealHumanUser) {
+      handleCaptchaResolved({
+        nativeEvent: {
+          data: 'special-captcha-bypass-token',
```

Comment:
@etuleu isE2EEnv is a dev only config, but as this is used to bypass the captcha and that does seem prone to abuse. @jeanregisser is this safe to do?

Reply:
This is alfajores, the possible abuse is that someone uses up komenci's funds to verify phone numbers on alfajores. I think the payout for abusing this is very small and the impact is also not that big since the funds the komenci address holds are limited.